code-blast.js

```diff
@@ -205,9 +205,9 @@ https://twitter.com/JoelBesada/status/670343885655293952
 		}
 
 		function onCodeMirrorChange(e) {
-			throttledShake(0.3);
-
 			setTimeout(function() {
+				if (!e) e = { start: editor.getCursorPosition() };
+				throttledShake(0.3);
```

Comment:
Generally it si good to avoid dom manipulation from the change event, since it can cause reflow making ace slow.
The shake is not delayed in a visible way.